ReactWindows/ReactNative.Shared/UIManager/ViewManager.cs

```diff
@@ -107,6 +108,12 @@ public void UpdateProperties(TFrameworkElement viewToUpdate, ReactStylesDiffMap
         public TFrameworkElement CreateView(ThemedReactContext reactContext)
         {
             var view = CreateViewInstance(reactContext);
+#if WINDOWS_UWP
+            if (view is UIElement uiElement)
+            {
```

Comment:
afaik - this check is redundant. TFrameworkElement is constrained to FrameworkElement, which is a derived class of UIElement.